### ISSUE TITLE
`sphinx-build clean` does not clean generated docs by automodapi

### ISSUE THREAD

**Issue by maxnoe** [astropy/sphinx-automodapi#156]
Is there a way to register "output" directories in extensions so that `sphinx-build clean` also cleans up the auto-generated output of automodapi?

**Comment by pllim** [comment]
Yes, you modify the directives under `clean:` in `docs/Makefile`.

**Comment by pllim** [comment]
That I am not sure, maybe @eteq knows.

FWIW if we decide to go ahead with https://github.com/astropy/astropy-sphinx-theme/issues/29 (use `pydata-sphinx-theme`), then we might not even need this anymore in the future.

**Comment by maxnoe** [comment]
How does the theme address auto generated docs?

**Comment by maxnoe** [comment]
Isn't autogenerating docs pages orthogonal to how the theme makes it look?

**Comment by pllim** [comment]
While we doing the theme change, I would also be interested to see how numpy and scipy generating their API... but I am not speaking for the project.
